openmetadata-service/src/main/java/org/openmetadata/csv/EntityCsv.java

```diff
@@ -275,20 +276,27 @@ protected final List<EntityReference> getEntityReferences(
   }
 
   protected final List<TagLabel> getTagLabels(
-      CSVPrinter printer, CSVRecord csvRecord, int fieldNumber) throws IOException {
+      CSVPrinter printer,
+      CSVRecord csvRecord,
+      List<Pair<Integer, TagSource>> fieldNumbersWithSource)
+      throws IOException {
     if (!processRecord) {
       return null;
     }
-    List<EntityReference> refs = getEntityReferences(printer, csvRecord, fieldNumber, Entity.TAG);
-    if (!processRecord || nullOrEmpty(refs)) {
-      return null;
-    }
     List<TagLabel> tagLabels = new ArrayList<>();
-    for (EntityReference ref : refs) {
-      tagLabels.add(
-          new TagLabel()
-              .withSource(TagSource.CLASSIFICATION)
-              .withTagFQN(ref.getFullyQualifiedName()));
+    for (Pair<Integer, TagSource> pair : fieldNumbersWithSource) {
+      int fieldNumbers = pair.getLeft();
+      TagSource source = pair.getRight();
+      List<EntityReference> refs =
+          pair.getRight() == TagSource.CLASSIFICATION
+              ? getEntityReferences(printer, csvRecord, fieldNumbers, Entity.TAG)
+              : getEntityReferences(printer, csvRecord, fieldNumbers, Entity.GLOSSARY_TERM);
+      if (!processRecord || nullOrEmpty(refs)) {
```

Comment:
can we inverse this if condition, instead of having a continue

Reply:
done